docs/src/modules/components/ApiPage.js

```diff
@@ -53,11 +53,13 @@ function Heading(props) {
 
   return (
     <Level id={hash}>
-      {getTranslatedHeader(t, hash)}
-      <a aria-labelledby={hash} className="anchor-link" href={`#${hash}`} tabIndex={-1}>
-        <svg>
-          <use xlinkHref="#anchor-link-icon" />
-        </svg>
+      <a aria-labelledby={hash} className="title-link-to-anchor" href={`#${hash}`} tabIndex={-1}>
+        {getTranslatedHeader(t, hash)}
+        <div className="anchor-icon">
```

Comment:
We can't do this: https://validator.w3.org/nu/?doc=https%3A%2F%2Fnext--material-ui.netlify.app%2Fmaterial-ui%2Fgetting-started%2F

<img width="515" alt="SCR-20240421-cero" src="https://github.com/mui/material-ui/assets/3165635/ccfba9c9-3788-48c5-a180-63c3706b3dac">

I guess this would fix the regression:

```suggestion
        <span className="anchor-icon">
```